eks-distro-base/minimal-image-rebuilder.sh

```diff
@@ -0,0 +1,12 @@
+set -e
```

Comment:
let's also add a comment to the top of this which talks about what it's for and why, for posterity sake